Added custom entity data parameters (#2007)
MCreator/MCreator: plugins/mcreator-localization/help/default/entity/data_default_value.md

> This is the default value of the entity's variable.

Reviewer: Задаёт значение по умолчанию этой переменной, в зависимости от типа:
\* \*\*Logic**: \_true_ или \_false_;
\* \*\*Number**: Одно из чисел в диапазоне от минимального до максимального разрешённых целочисленных значений;
\* \*\*String**: Обычный текст.

> - **Logic**: _true_ or _false_;

Reviewer 2: We usually use * not -, and don't split lines with semicolon in those lists in other tips

> - **Number**: It can go from the maximal negative limit to the maximal positive limit of an integer.
> - **Logic**: true or false
> - **String**: A default text

Reviewer: How can text be default?
```suggestion
- **Number**: Can go from the maximal negative limit to the maximal positive limit of an integer;
- **Logic**: _true_ or _false_;
- **String**: Simple text.
```

Author: > How can text be default?

I didn't know what to write for this one. xD

Author: Keep the original order, so they are ordered the same way as inside the list.

Reviewer: I thought this is like the default order of primitive types, similar to what was in procedures when creating a local variable.

Reviewer: Increasing in "complexity": boolean -> 2 values, integer -> 16^8 values, string -> ≈∞ values.
